examples/hooks/components/RefinementList.tsx

```diff
@@ -8,7 +8,10 @@ import { ControlledSearchBox } from './ControlledSearchBox';
 import { cx } from '../cx';
 
 export type RefinementListProps = React.ComponentProps<'div'> &
-  UseRefinementListProps;
+  UseRefinementListProps & {
+    searchable?: boolean;
+    searchablePlaceholder?: string;
```

Comment:
This should be in the IS.js types, right? Looks like we need to fix this at that level.

Reply:
those are the InstantSearch widget types, not the ones from the connector

Reply:
So we can use the IS.js widget type for this component?

Reply:
RefinementListWidgetParams contains unrelated types as well, are you suggesting `Pick<RefinementListWidgetParams, 'searchable' | 'searchablePlaceholder'>` ?